__tests__/app.test.js

```diff
@@ -315,3 +315,29 @@ describe("DELETE:200  /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("GET:200 Responds with : an array of objects, each object should have appropriate properties:", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(4);
+        body.forEach((topic) => {
+          expect(topic).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
+  test("GET:404 Responds with a 404 error message for incorrect path", () => {
```

Comment:
This would fall under a different path (general 404) so we do not need this test here